plugins/package-managers/node/src/funTest/assets/projects/synthetic/npm-babel-expected-output.yml

```diff
@@ -2245,7 +2245,7 @@ packages:
       algorithm: "SHA-1"
   vcs:
     type: "Git"
-    url: "git+https://github.com/es128/anymatch.git"
+    url: "https://github.com/es128/anymatch"
```

Comment:
Just an observation: Weird that the `.git` suffix sometimes gets stripped, and sometimes not.